espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/FindActive.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.handlers;
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
+import io.appium.espressoserver.lib.model.AppiumParams;
+import io.appium.espressoserver.lib.model.Element;
+import io.appium.espressoserver.lib.model.Locator;
+import io.appium.espressoserver.lib.model.Strategy;
+import io.appium.espressoserver.lib.model.ViewAttributesEnum;
+
+public class FindActive implements RequestHandler<AppiumParams, Element> {
+
+    @Override
+    public Element handle(AppiumParams params) throws AppiumException {
+        final Locator locator = new Locator();
+        locator.initUriMapping(params);
+        locator.setUsing(Strategy.XPATH);
+        locator.setValue(String.format("//*[@%s='true']", ViewAttributesEnum.FOCUSED.toString()));
```

Comment:
Maybe it's better to use  `View.isFocused()` instead this trick?

Reply:
It would generate much more code than this

Reply:
Few extra lines better than magic locators. Also performance will be increased significantly.